vm-setup/roles/virtbmc/tasks/setup_tasks.yml

```diff
@@ -52,13 +52,22 @@
     vbmc_libvirt_uri: "qemu+ssh://{{ non_root_user }}@{{ vbmc_address }}/session?socket=/run/user/{{ non_root_user_uid }}/libvirt/libvirt-sock&keyfile=/root/.ssh/id_rsa_virt_power&no_verify=1&no_tty=1"
   when: vbmc_libvirt_uri is not defined
 
-- name: Start the Virtual BMCs (virtualbmc >= 1.4.0+)
+- name: Start the Virtual BMCs (virtualbmc >= 1.4.0+) on CentOS
   service:
     name: "virtualbmc"
     state: started
     enabled: true
+  when:
+    - ansible_facts['distribution'] == "CentOS"
   become: true
 
+- name: Start the Virtual BMCs (virtualbmc >= 1.4.0+) on ubuntu
+  shell: vbmcd || true
```

Comment:
A service based approach would be better, or using a containerized vbmc

Reply:
Virtual BMCs service is not available on Ubuntu, so containerized version should be better in this case. We can propose this approach in another issue in the future. At the moment, I believe that we should show the community a runnable version of dev-env, and get their feedback as well.

Reply:
ok, up to you